packages/payment-detection/test/erc777/superfluid-retriever.test.ts

```diff
@@ -8,211 +8,105 @@ import { mockSuperfluidSubgraph } from './mocks';
-      );
-      const transferEvents = await graphRetriever.getTransferEvents();
-      expect(transferEvents).toHaveLength(1);
-      expect(transferEvents[0].amount).toEqual(paymentData.amount);
-      expect(transferEvents[0].name).toEqual('payment');
-      expect(transferEvents[0].parameters?.to).toEqual(paymentData.to);
-    });
-  });
+testSuite('rinkeby', '0x745861aed1eee363b4aaa5f1994be40b1e05ff90');
+testSuite('goerli', '0x2bf02814ea0b2b155ed47b7cede18caa752940e6');
 
+const testSuite2 = (network: string, token: string) => {
   describe('on ongoing request', () => {
-    it('should get payment event from SuperFluid via subgraph with ongoing request (Rinkeby)', async () => {
-      const paymentData = {
-        reference: '0xbeefac0e87b43bf1e99c82',
-        from: '0x165a26628ac843e97f657e648b004226fbb7f7c5',
-        to: '0xe7e6431f08db273d915b49888f0c67ef61802e05',
-        network: 'rinkeby',
-        salt: '0ee84db293a752c6',
-        amount: '1',
-        requestId: '0688792633ff0ec72a7dbdefb886d2db6cccfa98287320839c2f273c7a4e3ce7e2',
-        token: '0x0f1d7c55a2b133e000ea10eec03c774e0d6796e8', //fUSDCx
-        timestamp: 1643041225,
-      };
-      graphql.request.mockResolvedValue(mockSuperfluidSubgraph[2]);
-
-      const paymentReference = PaymentReferenceCalculator.calculate(
-        paymentData.requestId,
-        paymentData.salt,
-        paymentData.to,
-      );
-      const subgraphReference = `0xbeefac${paymentReference}`;
-      expect(subgraphReference).toEqual(paymentData.reference);
-      const graphRetriever = new SuperFluidInfoRetriever(
-        paymentReference,
-        paymentData.token,
-        paymentData.to,
-        PaymentTypes.EVENTS_NAMES.PAYMENT,
-        paymentData.network,
-      );
-      jest.spyOn(Date, 'now').mockImplementation(() => 1643126596704);
-      const transferEvents = await graphRetriever.getTransferEvents();
-      const timestamp = Math.floor(Date.now() / 1000) - paymentData.timestamp;
-      expect(transferEvents).toHaveLength(1);
-      expect(transferEvents[0].amount).toEqual(timestamp.toString());
-      expect(transferEvents[0].name).toEqual('payment');
-      expect(transferEvents[0].parameters?.to).toEqual(paymentData.to);
-    });
-
-    it('should get payment event from SuperFluid via subgraph with ongoing request (Goerli)', async () => {
+    it(`should get payment event from SuperFluid via subgraph with ongoing request on ${network}`, async () => {
       const paymentData = {
         reference: '0xbeefac0e87b43bf1e99c82',
         from: '0x165a26628ac843e97f657e648b004226fbb7f7c5',
         to: '0xe7e6431f08db273d915b49888f0c67ef61802e05',
-        network: 'goerli',
+        network: network,
         salt: '0ee84db293a752c6',
         amount: '1',
         requestId: '0688792633ff0ec72a7dbdefb886d2db6cccfa98287320839c2f273c7a4e3ce7e2',
-        token: '0x2bf02814ea0b2b155ed47b7cede18caa752940e6', //fDaix
+        token: token, //fUSDCx
```

Comment:
You call this function with 2 different tokens. fDAIx on Goerli and fUSDCx on Rinkeby.
it might be nice specify it. 
i.e: create 2 const, fDAIxToken and fUSDCxToken. and delete the comment here

Reply:
ok